### PR TITLE
Task 4 GET /api/articles/:article_id and tests done

### PR DESCRIPTION
**Pull request by kantideep**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -33,12 +44,41 @@ describe('1.1 GET api/topics', () => {
...
+                    votes: 100
+                });
+            });
+    });
+    test('400: returns an error when passed an ID of an invalid type', () => {
+        return request(app)
+            .get('/api/articles/banana')
+            .expect(400)
+            .then(({ body }) => {
+                console.log(body);
```
Comment (Dutton1990) [inline __tests__/app.test.js]: dont forget to remove console logs!

File: models/model.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        })
+};
+
+exports.selectArticleById = (article_id) => {
+
+    sqlQuery = `SELECT u.username, a.title, a.article_id, a.body, a.topic, a.created_at, a.votes
+                FROM users u, articles a
+                WHERE u.username = a.author
+                AND a.article_id = $1;`
+
```
Comment (Dutton1990) [inline models/model.js:21]: The seed should be giving you access to author on the article table so I don't think you need to do this. However it does work! Change it if you like or if you prefer this way, fine by me!

**Comment by Dutton1990** [comment]
there's a few console logs in there but otherwise this looks fine! just one suggestion but i'll leave that upto you!

File: __tests__/app.test.js (hunk 1 above)
Comment (kantideep) [reply __tests__/app.test.js]: I've removed the console.logs now.

File: models/model.js (hunk 2 above)
Comment (kantideep) [reply models/model.js:21]: I'll change it when I finish all the other tasks :-)

**Pull request merged**